docs.css

```diff
@@ -75,6 +75,9 @@ a {
 .ml-auto{
   margin-left: auto;
 }
+.font-30{
+  font-size: 30px;
+}
 #themeToggle {
   font-size: 1.5rem;
```

Comment:
You have used two different units of measurement. I feel this can be avoided.

Reply:
This is not a part of the CL only used for one specific use case.